docs/architecture-and-design.md

```diff
@@ -386,6 +390,20 @@ Most [Intents](#intents) are easy to break down into  [Primitives](#primitives),
 
 In order for a robot to move to the desired destination of a `MoveIntent`, the Navigator will use various path-planning algorithms to find a path across the field that does not collide with any robots or violate any restrictions set on the `MoveIntent`. The Navigator then translates this path into a series of `MovePrimitives`, which are sent to the robot sequentially so that it follows the planned path across the field.
 
+### Path Manager
+The `Path Manager` is responsible for generating a set of paths that don't collide. It is given a set of [Path Objective](#path-objective)s and [Path Planner](#path-planner), and it will generate paths using the given path planner and arbitrate between paths to prevent collisions. One approach is to add extra obstacles to cause the path planner to generate paths that avoid potential collisions; this is what `VelocityObstaclePathManager` implements. The approach is somewhat like using [Minkowski space](https://en.wikipedia.org/wiki/Minkowski_space), but where we assume that a robot will occupy all the positions along the path for the next time step. However, it's a very crude implementation.
```

Comment:
somewhat of a nit: I think the examples of how a `Path Manager` may choose to arbitrate is helpful, but mentioning the `VelocityObstaclePathManager` feels too close to talking about actual implementations. I think we can remote the "this is what `VelocityObstaclePathManager` implements" part.

Reply:
addressed